frontend/src/components/Home.js

```diff
@@ -44,7 +45,7 @@ const Mission = () => {
 const Value = ({value, description, img}) => {
     return (
         <div className="col-12 col-md-6 text-center">
-            <img src={img} height="148" alt=""></img>
+            <img src={img} height="148" alt="" class="rounded"></img>
```

Comment:
From the preview, I can see that the images are not in the same aspect ratio. Would be nice to have same ratio tho.